.github/workflows/practice-basic-webapi-application.yml

```diff
@@ -0,0 +1,19 @@
+# This workflow will build a .NET project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
```

Comment:
```suggestion
```
Don't forget to clean up boiler plate code. Look at everything before you request a review and make sure you clean up unnecessary comments.